src/external/google/protobuf/CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
-project(TuriExternalDependencies)
+project(Turi)
```

Comment:
what''s the reason to change here?

Reply:
This is the name of the generated Xcode project that these files are added to.  Thus all project specifications should be Turi if we generate an xcode project, which we can do here.

Reply:
I see.